tests/UI/campaigns/functional/BO/02_orders/01_orders/05_checkDeliverySlipDownloadedFromList.js

```diff
@@ -1,187 +0,0 @@
-require('module-alias/register');
```

Comment:
Why removing this test ?

Reply:
Because the new test 'Update status' regroups the tests 'Check invoice and check credit slip'